docs/src/main/paradox/play-framework.md

```diff
@@ -101,11 +101,13 @@ Scala
 Java
 :   @@snip[GreeterServiceImpl.java]($root$/../play-interop-test-java/src/main/java/controllers/GreeterServiceImpl.java) { #service-impl }
 
-And then add this to your Play `conf/routes` file, note that the path here depends on the package and service name from the
-`.proto` service descriptor and cannot be set to an arbitrary value (if you try to do so an exception will be thrown when the router is started).
+And then add the router to your Play `conf/routes` file. Note that the router alerady knows its own path since it is
```

Comment:
"already"

Reply:
"And then add the router" - is it a "router"? Isn't it the "controller"?

Reply:
Hmm I think it is a Router... not sure `public class GreeterServiceImpl extends AbstractGreeterServiceRouter {` in Play wording I think Router is a lower level thing simply hm